apps/mobile/src/components/blurred-header.tsx

```diff
@@ -21,6 +21,61 @@ type NativeStackHeaderProps = Parameters<
 
 export const HEADER_HEIGHT = 64;
 
+export function createSimpleHeader(props: {
+  left?: ReactNode;
+  center?: ReactNode;
+  right?: ReactNode;
+  insets: EdgeInsets;
+}) {
```

Comment:
Why not just a component accepting these props? Not clear why this needs to be a HOC

Reply:
isn't that just the difference between `<SomeComponent />` and `() => <SomeComponent />`?